source/includes/_usecases.md

```diff
@@ -94,8 +94,13 @@ The [Affiliates Integration Guide](#affiliates-integration-guide) helps you get
 
 ## Receive Money
 
-You can receive money to the local bank details that come with your TransferWise account (AUD, EUR, GBP, NZD and USD) and reconcile these incoming payments via the API.
+You can receive money to the local bank details that come with your TransferWise account (USD, EUR, GBP, USD, NZD and PLN) and reconcile these incoming payments via the API.
 
+You’re also able to create a webhook subscription to receive information about incoming payments to your server. Here’s more information about the [webhooks](#webhook-events).
```

Comment:
Slightly off wording, the information is going to your server, not the payments! Suggest:
> You’re also able to create a webhook subscription to receive event callbacks to your server when payments are received.